### PR TITLE
Remove unused imports of files under the directory of layers and applications

### PR DESCRIPTION
**Pull request by kemaswill**
Remove unused imports of files under the directory of layers and applications.

### CODE REVIEW

File: keras/applications/inception_v3.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 from ..utils.layer_utils import convert_all_kernels_in_model
 from ..utils.data_utils import get_file
 from .. import backend as K
-from .imagenet_utils import decode_predictions, _obtain_input_shape
```
Comment (fchollet) [inline keras/applications/inception_v3.py]: This should be kept (for namespace consistency: you want to be able to import `inception_v3.decode_predictions`).

File: keras/applications/music_tagger_crnn.py [hunk 2]
```diff
@@ -21,7 +20,6 @@
 from ..engine.topology import get_source_inputs
 from ..utils.data_utils import get_file
 from ..utils.layer_utils import convert_all_kernels_in_model
-from .audio_conv_utils import decode_predictions, preprocess_input
```
Comment (fchollet) [inline keras/applications/music_tagger_crnn.py]: Same

File: keras/applications/resnet50.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 from ..engine.topology import get_source_inputs
 from ..utils.layer_utils import convert_all_kernels_in_model
 from ..utils.data_utils import get_file
-from .imagenet_utils import decode_predictions, preprocess_input, _obtain_input_shape
```
Comment (fchollet) [inline keras/applications/resnet50.py]: Same

File: keras/applications/vgg16.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
 from ..utils.layer_utils import convert_all_kernels_in_model
 from ..utils.data_utils import get_file
 from .. import backend as K
-from .imagenet_utils import decode_predictions, preprocess_input, _obtain_input_shape
```
Comment (fchollet) [inline keras/applications/vgg16.py]: Same

File: keras/applications/vgg19.py [hunk 5]
```diff
@@ -18,7 +18,7 @@
 from ..utils.layer_utils import convert_all_kernels_in_model
 from ..utils.data_utils import get_file
 from .. import backend as K
-from .imagenet_utils import decode_predictions, preprocess_input, _obtain_input_shape
```
Comment (fchollet) [inline keras/applications/vgg19.py]: Same

File: keras/applications/xception.py [hunk 6]
```diff
@@ -28,7 +28,7 @@
 from ..engine.topology import get_source_inputs
 from ..utils.data_utils import get_file
 from .. import backend as K
-from .imagenet_utils import decode_predictions, _obtain_input_shape
```
Comment (fchollet) [inline keras/applications/xception.py]: Same

**Comment by kemaswill** [comment]
I see.

**Pull request closed without merging**